pennylane/variable.py

```diff
@@ -137,10 +140,11 @@ def val(self):
         # pylint: disable=unsubscriptable-object
         if self.name is None:
             # The variable is a placeholder for a positional argument
-            return self.free_param_values[self.idx] * self.mult
+            return Variable.free_param_values[self.idx] * self.mult
```

Comment:
Are there any side effects to this change?

Reply:
There shouldn't be, both should be referring to the class variable, the new version is explicit about it.